tools/pipelines/build-azure.yml

```diff
@@ -93,10 +93,10 @@ pr:
     - packages/framework/loader/container-loader
     - packages/test/test-end-to-end-tests
     - tools/pipelines/test-service-clients.yml
-    - tools/pipelines/templates/include-test-real-service.yml
+    - tools/pipelines/templates/1ES/include-test-real-service.yml
```

Comment:
Let's actually close this one. I just disabled the `Build - azure` pipeline in both public and internal projects. Since we moved the azure client packages to the client release group this pipeline has no longer been in use. We should delete the file at some point.

Also, not sure why it had a trigger dependency on `ìnclude-test-real-service.yml`, it doesn't use it.